package.json

```diff
@@ -24,6 +24,7 @@
     "test:primitives": "jest -c .jest.primitives.json",
     "test:primitives:watch": "npm run test:primitives -- --watch",
     "test:integration": "jest -c .jest.integration.json --forceExit",
+    "test:jest-styled-components": "npm install jest-styled-components@next && npm run test:integration",
```

Comment:
Should we add `jest-styled-components` to the `devDependencies` instead? I think running `npm install` here if people use `yarn` will take a looong time

Reply:
Thank you very much for your comment, @mxstbr.

As far as I know, there's no way to install `devDependencies` without adding them to the `yarn.lock`.
What I was trying to do here is running the tests against the latest version of `jest-styled-components` (and avoid keeping it manually updated).
I'm not entirely sure why this should be slower for `yarn` users (this project is actually using `yarn`) but since the test doesn't need to be run all the time I wouldn't mind about perf.

That said, I'm more than happy to add the package to the `devDependencies`, stick to a particular version, and keep the dependency updated.

Reply:
Can't we do something like

```
devDependencies: {
  "jest-styled-components": "*"
}
```

Reply:
Unless I'm missing something, that's going to add something like this to the `yarn.lock`:

```diff
+jest-styled-components@*:
+  version "4.10.0"
+  resolved "https://registry.yarnpkg.com/jest-styled-components/-/jest-styled-components-4.10.0.tgz#6fcd83208fefc4ceb87b79b55d44697a05d226a8"
+  dependencies:
+    css "^2.2.1"
+
```

Reply:
Ah god dammit, I forgot about the lock file. Ugh, that sucks. I see what you mean now, that's fine then, sorry for the confusion.